src/document/mod.rs

```diff
@@ -200,28 +360,6 @@ pub trait DocumentOps {
 
 #[async_trait]
 impl DocumentOps for crate::IronOxide {
-    async fn document_list(&self) -> Result<DocumentListResult> {
```

Comment:
What are these changes? Reordering maybe?

Reply:
Yes, re-ordering. The order we put in the file changes the order of the docs, so I tried to make a logical progress of functions that started with encrypt/decrypt